std/portable/index.js

```diff
@@ -198,11 +198,38 @@ globalScope["changetype"] = function changetype(value) {
 };
 
 String["fromCharCodes"] = function fromCharCodes(arr) {
-  return String.fromCharCode.apply(String, arr);
+  const CHUNKSIZE = 1 << 14;
```

Comment:
I remember that I did `1 << 13` there in the past, but I don't remember exactly how I figured out this limit. Not a huge difference, but perhaps staying conservative here is worth it so we don't run into issues if this function is called in a code path already quite deep into the stack. Wdyt?

Reply:
I'm ok with `1 << 13` as well. It's not big deal.

Reply:
Not sure why, but on latest node (not lts) new chunked approach slowdown tests. Try to increase `1 << 13` to `1 << 15`